cmd/gapit/trim_initial_state.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"io/ioutil"
+
+	"github.com/google/gapid/core/app"
+	"github.com/google/gapid/core/log"
+)
+
+type trimInitialStateVerb struct{ TrimInitialStateFlags }
+
+func init() {
+	verb := &trimInitialStateVerb{}
+	app.AddVerb(&app.Verb{
+		Name:      "trim_initial_state",
+		ShortHelp: "(WIP) Trims a gfx trace's initial state to the resources actually used by the commands",
```

Comment:
Remove the WIP part. `gapit` is not meant for the faint of heart.

Reply:
done